src/errors/types.rs

```diff
@@ -54,7 +54,9 @@ pub enum ErrorType {
     #[strum(message = "Field required")]
     Missing,
     #[strum(message = "Field is frozen")]
-    Frozen,
+    FrozenField,
+    #[strum(message = "Model is frozen")]
+    FrozenModel,
```

Comment:
Might be nice if this can work for both models and dataclasses, therefore let's go with an ambiguous name
```suggestion
    #[strum(message = "Instance is frozen")]
    InstanceModel,
```

This also most closely matches what dataclasses uses:

```
dataclasses.FrozenInstanceError: cannot assign to field 'v'
```

Reply:
```suggestion
    #[strum(message = "Instance is frozen")]
    FrozenInstance,
```
right?